docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md

```diff
@@ -13,6 +13,54 @@ The General Data Protection Regulation (GDPR) is a European regulation to harmon
+
+### Data backups
+
+OpenProject cloud environment is continuously backing up user data with data at rest being fully encrypted with AES-256. Each individual instance is logically separated and data is persisted in a unique database schema, reducing the risk of intersection or data leaks between instances.
+
+### Access **to** data and infrastructure
+
+Production infrastructure is accessible only for a strict set of authorized system operations personnel from a secure internal maintenance VPN. Services employed by employees are secured by Two-factor-authentication where available. Access to customer data is performed only when requested by the customer (i.e., as part of a support or data import/export request).
+
+All OpenProject GmbH employees employ industry standard data security measurements to secure their devices and access to cloud and on-premises infrastructure. All sensitive user data on laptops and workstations are encrypted and machines are maintained to receive system updates.
+
+## Data Management and Portability
+
+The GDPR includes grants to every data subject the right to access, modify, receive, and delete their own data. 
+
+OpenProject customers with admin accounts on their instance act as data controllers for their team members and have elaborate means to perform these request on behalf of the data subjects they are responsible for.
+
+We detail some of these rights of the data subject in the following segments.
+
+### Right to Access and Rectification
+
+With OpenProject, data controllers have fine-grained user and rights management to perform these requests. Individual data subjects can forward any request to their responsible data controller of their information.
+
+The following resources provide additional information:
+
+- [Managing accounts and users in your instance](../../../system-admin-guide/users-permissions/) (for data controllers).
+
+### Right to Erasure (“Right to be forgotten”)
+
+OpenProject provides means to fully erase both all identifiable information of a user from the application. If the user is still referenced from data within the instance, these references are replaced with an anonymous user to ensure the data integrity of the application.
+
+- Data controllers can perform the deletion [through the administration](../../../system-admin-guide/users-permissions/users/).
+
+- Depending on the configuration of your OpenProject instance, individual data subjects may perform the deletion of their own account through the [Delete Account](../../../getting-started/my-account/) page. If this is disabled, the request may be stated to the data controller.
+
+### Data Portability
+
+OpenProject provides means to data controllers in order to receive *all* personal data connected to the OpenProject instance. This encompasses all user and system data (in the form of an SQL dump) as well as a collection of all uploaded files.
```

Comment:
This is now possible by controllers on their own using the backup feature of OpenProject: https://www.openproject.org/docs/enterprise-guide/enterprise-cloud-guide/backups/

Could you add that to the statement?